src/docs/asciidoc/jcache/icache.adoc

```diff
@@ -509,6 +509,8 @@ by `com.hazelcast.cache.ICache` featuring the `ExpiryPolicy` parameter:
 
 Asynchronous method overloads are not listed here. Please see <<icache-async-methods, ICache Async Methods>> for the list of asynchronous method overloads.
 
+ICache also offers `setExpiryPolicy` method to associate certain keys with custom expiry policies.
+Per key expiry policies defined by this method takes precedence over cache policies but they are overridden by expiry policies specified in above mentioned overloaded methods.
```

Comment:
Per key expiry policies defined by this method take precedence over cache policies, but they are overridden by the expiry policies specified in above mentioned overloaded methods.